### PR TITLE
fix tests and separate UI test from unittest schema

### PR DESCRIPTION
**Pull request by huminzhi**

### CODE REVIEW

**Comment by khawkins** [comment]
Seems fine to me.

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
-[![Build Status](https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS.svg?branch=unstable)](https://travis-ci.org/forcedotcom/SalesforceMobileSDK-iOS)
```
Comment (bhariharan) [inline readme.md]: Should we wait till we have a `Cloudbees` link before we remove this one?

**Comment by huminzhi** [comment]
@bhariharan I found a way to work around this issue, will update pr soon.. thanks

**Comment by huminzhi** [comment]
something bad happens to my branch, close it now

**Pull request closed without merging**